src/app/test-utils/map-matchers.spec.ts

```diff
@@ -0,0 +1,21 @@
+export const mapMatchers: jasmine.CustomMatcherFactories = {
+  toBeKeyOf: (util) => {
```

Comment:
```suggestion
  toHaveKey: (util) => {
```
Lets turn this matchers around to be more in line with other matchers like `toContain` or `toHaveOwnProperty`